<?xml version="1.0"?>
<launch>

    <node name="diff_drive_mppi" pkg="ccv_mppi_path_tracker" type="diff_drive_mppi" output="screen">
    <!-- <param name="mppi_path_tracker_config" textfile="$(find ccv_mppi_path_tracker)/config/diff_drive_mppi_path_tracker.yaml"/> -->
    <param name="v_ref" value="1.2"/>
    <param name="v_max" value="1.0"/>
    </node>
    <node name="reference_path_creater" pkg="ccv_mppi_path_tracker" type="reference_path_creater" output="screen">
    <param name="world_frame_id" value="odom"/>
    <param name="A1" value="1"/>
    <param name="omega1" value="1"/>
    <remap from="/reference_path_creater/path" to="/reference_path"/>
    </node>

    <node name="rviz" pkg="rviz" type="rviz" args="-d $(find ccv_mppi_path_tracker)/config/config.rviz" output="screen"/>
</launch>
